runners/spark/3/src/main/java/org/apache/beam/runners/spark/structuredstreaming/SparkStructuredStreamingRunner.java

```diff
@@ -222,7 +225,7 @@ private static Future<?> runAsync(Runnable task) {
     return future;
   }
 
-  private static @Nullable Runnable stopSparkSession(SparkSession session, boolean isProvided) {
+  private static @Nullable Runnable sparkStopFun(SparkSession session, boolean isProvided) {
```

Comment:
nit: what `Fun` means in this context?

Reply:
`Function`, I'll rename to `sparkStopFn` 👍